Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM golang:1.16.5-alpine3.12 as builder
 WORKDIR /go/src/github.com/mendersoftware/iot-manager
+RUN mkdir -p /etc_extra
+RUN echo "nobody:x:65534:" > /etc_extra/group
+RUN echo "nobody:!::0:::::" > /etc_extra/shadow
+RUN echo "nobody:x:65534:65534:Nobody:/:" > /etc_extra/passwd
+RUN chown -R nobody:nobody /etc_extra
```

Comment:
my intention was to ensure all the files in the final image are owned by `nobody`, but since there is no user info in the scratch image, the `--chown` way is not an option and I had to take the detour with this... in order to run the image this _should_ not make a difference afaik, but I'm all ears if you think this could cause any problems 👍